indra/sources/dgi/__main__.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+"""Command line interface for DGI-DB."""
+
+from collections import Counter
+
+from tabulate import tabulate
```

Comment:
Is tabulate an indirect dependency through some other dependency of INDRA (maybe pandas)?

Reply:
No, unfortunately it's not. This is a one-off summary script, so I could also delete it from the PR. Otherwise, if someone wants to run it, I think it would be obvious enough to `pip install tabulate` from the import warning